plugins/shared-react/src/components/pipeline/HorizontalStackedBars.tsx

```diff
@@ -31,6 +31,7 @@ export const HorizontalStackedBars = ({
 }: HorizontalStackedBarsProps) => (
   <div
     id={`horizontal-stacked-bars-${id}`}
+    data-testid={`horizontal-stacked-bars-${id}`}
```

Comment:
@debsmita1 should we get changes for plugin `shared-react` in seperate PR/commit? and this should update only tekton plugin?

Reply:
Created this https://github.com/janus-idp/backstage-plugins/pull/628

Reply:
Thanks , have merged it so can rebase.